nixos/modules/services/x11/display-managers/gdm.nix

```diff
@@ -168,7 +168,8 @@ in
     systemd.tmpfiles.rules = [
       "d /run/gdm/.config 0711 gdm gdm -"
     ] ++ optionals config.hardware.pulseaudio.enable [
-      "L+ /run/gdm/.config/pulse - - - - ${pulseConfig}"
+      "d /run/gdm/.config/pulse 0711 gdm gdm"
```

Comment:
this is different than `"d /run/gdm/.config 0711 gdm gdm -"`

Reply:
Omitting the age filed has the same effect as `-` according to the [man page](https://www.freedesktop.org/software/systemd/man/tmpfiles.d.html).
Or are you referring to something else?

Reply:
sorry, https://github.com/NixOS/nixpkgs/pull/80274/files#r379931503 for consistency